subcommands/common.go

```diff
@@ -164,3 +164,14 @@ func PrintConfigs(deviceConfigs []client.DeviceConfig, listLimit int) int {
 	}
 	return listLimit
 }
+
+func IntersectionInSlices(list1, list2 []string) bool {
```

Comment:
That's really interesting to fidn out that golang (base modules) don't include such basic functionality.